packages/core/src/tasks/index.ts

```diff
@@ -738,6 +747,12 @@ export const chugsplashDeployAbstractTask = async (
 
   // At this point, the deployment has been completed.
   if (integration === 'hardhat') {
+    if (!(await isLiveNetwork(provider))) {
+      // We save the snapshot ID here so that tests on the stand-alone Hardhat network can be run
+      // against the most recently deployed contracts.
+      await writeSnapshotId(provider, networkName, deploymentFolder)
+    }
```

Comment:
Needed to add this so that tests on the stand-alone Hardhat network work properly after upgrading a proxy via the chugsplash-deploy task.